src/Book.js

```diff
@@ -0,0 +1,72 @@
+import React, {useState, useEffect} from 'react';
+import './App.css';
+import {loadJson} from './sendJson';
+import {useParams} from 'react-router-dom';
+
+const loadOneBook = async ({id}, onLoad, setError) => {
```

Comment:
id should not be destructured here. It should be destructured when you call `useParams`